app/controllers/submissions_controller.rb

```diff
@@ -317,6 +317,7 @@ def view
        !@submission.autograde_file.nil?
 
       file = @submission.autograde_file.read || "Empty Autograder Output"
+      is_autograder_output = true
```

Comment:
Seems like this variable `is_autograder_output` is initialized only if it is `true`?
If so we are relying on the default behavior of `unless` in line 497 when it is uninitialized, which I think might be risky. Lmk what you think

Reply:
imo, it's not really risky since the default behavior is well-defined and intuitive. But I've refactored the code to make the logic more obvious.